edgelet/edgelet-http-mgmt/src/server/mod.rs

```diff
@@ -71,6 +77,12 @@ impl ManagementService {
             put     Version2018_06_28 runtime Policy::Module(&*AGENT_NAME)  => "/identities/(?P<name>[^/]+)"        => UpdateIdentity::new(identity.clone()),
             delete  Version2018_06_28 runtime Policy::Module(&*AGENT_NAME)  => "/identities/(?P<name>[^/]+)"        => DeleteIdentity::new(identity.clone()),
 
+            get     Version2020_07_22 runtime Policy::Module(&*AGENT_NAME)  => "/modules/(?P<name>[^/]+)/secrets/(?P<id>[^/]+)" => GetSecret::new(secret.clone()),
```

Comment:
We don't need to add GetSecret, SetSecret and DeleteSecret because those are exposed in workload api. 
Where is RefreshSecret called?

Reply:
I will remove `{Get,Set,Delete}Secret` from here in that case.

`RefreshSecret` is not called. It will be necessary for direct method support in the future, however.